src/config.rs

```diff
@@ -262,6 +262,10 @@ pub struct StratusConfig {
     #[arg(long = "max-connections", env = "MAX_CONNECTIONS", default_value = "200")]
     pub max_connections: u32,
 
+    /// JSON-RPC max active subscriptions per client.
+    #[arg(long = "max-subscriptions", env = "MAX_SUBSCRIPTIONS", default_value = "10")]
```

Comment:
Here the default value still 10.

In another PR we should extract this to a separete struct RpcServerConfig.

Reply:
Mismatch fixed. Will take this refactoring in a next PR.